openapi/wicketd.json

```diff
@@ -0,0 +1,349 @@
+{
```

Comment:
Could we add an integration test that confirms this file stays in sync? The MGS one is at https://github.com/oxidecomputer/omicron/blob/adefcef27cc074f7c55cc4c0f12adee49eac3e5b/sled-agent/tests/integration_tests/commands.rs#L58-L79

Reply:
Done in [f3aa8da](https://github.com/oxidecomputer/omicron/pull/1930/commits/f3aa8da39612614079ee2bf6d03ec179d0489e39)